pkg/app/run.go

```diff
@@ -620,13 +620,19 @@ func commandRequiresAuthServer(command string) bool {
 
 // commandRequiresToken determines if the command to be executed is one that
 // requires an API token.
-func commandRequiresToken(command string) bool {
-	switch command {
-	case "compute build", "compute hash-files", "compute init", "compute metadata", "compute serve":
+func commandRequiresToken(command argparser.Command) bool {
+	commandName := command.Name()
+	switch commandName {
+	case "compute init":
+		if initCmd, ok := command.(*compute.InitCommand); ok {
+			return compute.IsFastlyID(initCmd.CloneFrom)
+		}
+		return false
```

Comment:
This really shouldn't happen, it would mean a second `compute init` command was registered somewhere. It might be worth `panic`ing here but that also seems very low-likelihood

Reply:
Agreed. I think this is fine. I'd rather avoid a panic and it's very unlikely that we'd encounter the command being registered twice.